MiniApp/Classes/core/Display/RealMiniAppView.swift

```diff
@@ -192,6 +192,19 @@ internal class RealMiniAppView: UIView {
             default:
                 if requestURL.isMiniAppURL(customMiniAppURL: miniAppURL) {
                     return decisionHandler(.allow)
+                } else if requestURL.isBase64 {
+                    if
+                        let miniAppId = miniAppId,
+                        let _ = MiniApp.shared()
+                            .getCustomPermissions(forMiniApp: miniAppId)
+                            .filter({ $0.permissionName == .attachments && $0.isPermissionGranted == .allowed })
+                            .first
+                    {
+                        self.navigationDelegate?.miniAppNavigation(shouldOpen: requestURL, with: { (url) in
+                            self.webView.load(URLRequest(url: url))
+                        })
```

Comment:
Guess we need to do bit of formatting here.

Reply:
also, instead of `let _ =` please use `!= nil`